### PR TITLE
Create effective-altruism-wiki.csl

### PR DESCRIPTION
**Pull request by benthamite**
Citation style for the [Effective Altruism Wiki](https://forum.effectivealtruism.org/tag/ea-wiki-faq).

### CODE REVIEW

**Comment by bwiernik** [comment]
Thanks! Can you add a template link to the style you based this off of, and also a link to the documentation for the style?

**Comment by adam3smith** [comment]
@retorquere -- sorry to trouble you one more time, but this shouldn't fail, I think. Looks like it's looking for Sheldon in the wrong path (3.0.0 instead of 3.0.2) -- is there a wrong ruby version somewhere?

**Comment by retorquere** [comment]
No trouble. Something must have changed in the environment sheldon runs in, as it is just flaming out entirely. I'm looking into it.

**Comment by retorquere** [comment]
OK, so sheldon works again, but this ties into the issue that it really is in need of retirement.

File: effective-altruism-wiki.csl [hunk 1]
```diff
@@ -0,0 +1,1167 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="1" initialize="true" page-range-format="expanded" demote-non-dropping-particle="never">
```
Comment (bwiernik) [inline effective-altruism-wiki.csl:2]: Please run your style through https://formatter.citationstyles.org/ to fix the indenting

File: effective-altruism-wiki.csl [hunk 2]
```diff
@@ -0,0 +1,1167 @@
...
+    <terms>
+      <term name="editortranslator" form="short">
+        <single>ed. &amp; trans.</single>
+        <multiple>eds. &amp; trans.</multiple>
+      </term>
+      <term name="translator" form="short">trans.</term>
+      <term name="interviewer" form="short">interviewer</term>
+      <term name="circa" form="short">ca.</term>
+      <term name="collection-editor" form="short">series ed.</term>
+      <term name="et-al"> /et al./</term>
```
Comment (bwiernik) [inline effective-altruism-wiki.csl:29]: Is ` /et al./` org mode syntax?

**Review by bwiernik — CHANGES_REQUESTED** [review]

**Comment by bwiernik** [comment]
Thanks! A few comments to start. This is based on APA 6 correct?

**Comment by benthamite** [comment]
My sincere apologies: I now see that I submitted the wrong file. I have two csl files: a markdown version and an org-mode version. I intended to submit the former, which had been run through the formatter and had all the words fully spelled out, but I made a mistake and submitted the org-mode version instead. Should I create a new pull request?

I'm sorry, again, for wasting your time.

**Comment by adam3smith** [comment]
You can just switch it out in this PR

**Comment by bwiernik** [comment]
It's no time waste at all. We're just doing some basic review of the style

**Comment by adam3smith** [comment]
No concerns from me. @bwiernik I'll let you merge once you're reviewed

File: effective-altruism-wiki.csl [hunk 3]
```diff
@@ -0,0 +1,1167 @@
...
+          </group>
+        </group>
+      </else-if>
+      <else-if type="book">
+        <choose>
+          <if variable="container-title">
+            <text macro="container-booklike"/>
+          </if>
+        </choose>
+      </else-if>
```
Comment (bwiernik) [inline effective-altruism-wiki.csl:998]:
```suggestion
```

File: effective-altruism-wiki.csl [hunk 4]
```diff
@@ -0,0 +1,1167 @@
...
+        <else>
+          <label variable="locator" form="short"/>
+        </else>
+      </choose>
+      <text variable="locator" prefix=" "/>
+    </group>
+  </macro>
+  <macro name="container">
+    <choose>
+      <if type="article article-journal article-magazine article-newspaper review review-book" match="any">
```
Comment (bwiernik) [inline effective-altruism-wiki.csl:945]:
```suggestion
      <if type="article-journal article-magazine article-newspaper review review-book" match="any">
```

File: effective-altruism-wiki.csl [hunk 5]
```diff
@@ -0,0 +1,1167 @@
...
+                        <text term="volume" form="short" font-style="italic" text-case="capitalize-first"/>
+                        <text variable="volume" font-style="italic"/>
+                      </group>
+                    </if>
+                  </choose>
+                </group>
+              </else>
+            </choose>
+          </group>
+          <group delimiter=", " suffix=", ">
```
Comment (bwiernik) [inline effective-altruism-wiki.csl:1064]:
```suggestion
          <group delimiter=", " prefix=", ">
```

**Review by bwiernik — CHANGES_REQUESTED** [review]
A few minor suggestions and corrections

**Comment by bwiernik** [comment]
Thanks!

**Pull request merged**